msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -0,0 +1,158 @@
+
+> [!IMPORTANT]
+> If you have integrated a tab with your meeting, then your app must follow the Teams [single sign-on (SSO) authentication flow](../tabs/how-to/authentication/auth-aad-sso.md) for tabs.
+
+> [!NOTE]
+> * Mobile clients support tabs only in pre and post meeting stages. The in-meeting experiences that is in-meeting dialog and panel are currently not available on mobile.
+> * Apps are supported only in private scheduled meetings.
+
+#### Pre-meeting app experience
+
+With the pre-meeting app experience, you can find and add meeting apps and perform pre-meeting tasks, such as developing a poll to survey meeting participants.
+
+**To add tabs to an existing meeting**
+
+1. In your calendar, select a meeting to which you want to add a tab.
+1. Select the **Details** tab and select plus <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/>. The tab gallery appears.
+
+    ![Pre-meeting experience](../assets/images/apps-in-meetings/PreMeeting.png)
+
+1. In the tab gallery, select the app that you want to add and follow the steps as required. The app is installed as a tab.
+
+    ![Pre-meeting tab view](../assets/images/apps-in-meetings/PreMeetingTab.png)
+
+    > [!NOTE]
+    > * You can also add a tab using the meeting **Chat** tab in an existing meeting.
+    > * Tab layout must be in an organized state if there are more than ten polls or surveys.
+
+#### In-meeting app experience
+
+With the in-meeting app experience, you can engage participants during the meeting by using apps or taking polls. Meeting apps are hosted in the top upper bar of the chat window as an in-meeting tab. When users add a tab to a meeting through the tab gallery, apps that are in-meeting experiences are displayed. Use the in-meeting dialog box to showcase actionable content for meeting participants. For more information, see [create apps for Teams meetings](create-apps-for-teams-meetings.md).
+
+**To use tabs during a meeting**
+
+1. After entering the meeting, from the top upper bar of the chat window, select the app you want to use. An app is visible in a Teams meeting in the side panel or the in-meeting dialog box.
+1. In the in-meeting dialog box, enter your response as a feedback.
+
+    ![Dialog box view](../assets/images/apps-in-meetings/in-meeting-dialog-view.png)
+
+    > [!NOTE]
+    > * Apps can leverage the Teams Client SDK to access the `meetingId`, `userMri`, and `frameContext` to render the experience appropriately.
+    > * If the result of a survey or poll is exported, you are notified that the results successfully downloaded.
+    > * Your app manifest specifies that your tab is [optimized for side panel](enable-and-configure-your-app-for-teams-meetings.md#during-a-meeting), that is where it is displayed. It can also be part of a share-tray experience, subject to specified design guidelines.
```

Comment:
Reword:

Your app manifest specifies the places where you want them to appear. The context field is used for this purpose. It can also be part of a share-tray experience, subject to specified design guidelines